src/encoded/static/components/search.js

```diff
@@ -1145,7 +1145,7 @@ class Facet extends React.Component {
         // Sort numerical terms by value not by frequency
         // This should ultimately be accomplished in the back end, but the front end fix is much simpler so we are starting with that
         // We have to check the full list for now (until schema change) because some lists contain both numerical and string terms ('Encyclopedia version' under Annotations) and we do not want to sort those by value
-        const numericalTest = a => !Number.isNaN(a.key);
+        const numericalTest = a => !isNaN(a.key);
```

Comment:
I remember trying to fix this linting error and it not working and reverting. I should have turned off the error or made a note of this!